extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -266,6 +267,11 @@ export class JupyterSession implements nb.ISession {
 				code: doNotCallChangeEndpointParams
 			}, true);
 			await future.done;
+
+			future = this.sessionImpl.kernel.requestExecute({
+				code: `%%configure -f${EOL}{"conf": {"spark.pyspark.python": "python3"}}`
```

Comment:
I'm fine with this for now, but we may want to consider exposing this via a setting.

Reply:
Python 2 reaches EOL in January, so we wouldn't get much use out of it.

@kevcunnane Your thoughts here?

Reply:
Also Spark is dropping support for Python 2. https://spark.apache.org/news/plan-for-dropping-python-2-support.html

Reply:
I'm OK hard coding given the EOL decision here. If we get a user request to support Python2 for a different target in the future we can look into a setting, but for now this is a good solution.